crates/web-plugins/didcomm-messaging/src/lib.rs

```diff
@@ -1,6 +1,6 @@
 mod manager;
 mod midlw;
-mod protocols;
+mod protocol;
```

Comment:
why did you rename from `protocols` to `protocol`?

Reply:
There were two folders of the same name.

Reply:
> There were 1 folders and a file of the same name.

There should be only one module with that name. Where does the other one come from?